### PR TITLE
adding opensmile

### PR DESCRIPTION
**Pull request by Rahul-Brito**
in progress, added a function but need to have it:

- take in Audio
- to Processing_signal

### CODE REVIEW

File: src/b2aiprep/process.py [hunk 1]
```diff
@@ -131,6 +131,19 @@ def resample_iir(audio: Audio, lowcut: float, new_sample_rate: int, order: int =
...
+    filename: 'str', feature_level: str = 'func'
+) -> torch.tensor: #feature_set: opensmile.FeatureSet = opensmile.FeatureSet.eGeMAPSv02
+    
+    smile = opensmile.Smile(
+        feature_set=opensmile.FeatureSet.eGeMAPSv02, #or path to conf: 'gemaps/eGeMAPSv02.conf'
+        feature_level=feature_level,
+        verbose=True,
+    )
+    
+    return smile.process_files(filename)
```
Comment (satra) [inline src/b2aiprep/process.py:144]: could we adjust this to Smile's audio-based method so we can use the Audio object? 

also can we parameterize the config set?

File: src/b2aiprep/process.py [hunk 2]
```diff
@@ -131,6 +131,19 @@ def resample_iir(audio: Audio, lowcut: float, new_sample_rate: int, order: int =
...
+
+
```
Comment (satra) [inline src/b2aiprep/process.py:146]: can we also add to `to_features`?

**Comment by Rahul-Brito** [comment]
agree on both points

**Comment by Rahul-Brito** [comment]
Implemented the above @satra. Not sure the protocol to merge, can I merge my own PR?

**Pull request merged**